module_utils/network_lsr/argument_validator.py

```diff
@@ -1858,6 +1919,20 @@ def _validate_post(self, value, name, result):
     VALIDATE_ONE_MODE_NM = "nm"
     VALIDATE_ONE_MODE_INITSCRIPTS = "initscripts"
 
+    def validate_route_tables(self, connection, idx):
+        iproute = IPRouteUtils()
+        for r in connection["ip"]["route"]:
+            if isinstance(r["table"], Util.STRING_TYPE):
+                table_cache = iproute.get_route_tables_mapping()
```

Comment:
`table_cache` is not a good name. Which table? What significance has the "cache" here? Call it what is is. For example, `mapping`.